packages/@react-types/button/src/index.d.ts

```diff
@@ -48,13 +48,13 @@ export interface LinkButtonProps<T extends ElementType = 'button'> extends AriaB
 
 interface AriaBaseButtonProps extends FocusableDOMProps, AriaLabelingProps {
   /** Indicates whether the element, or another grouping element it controls, is currently expanded or collapsed. */
-  'aria-expanded'?: boolean,
+  'aria-expanded'?: boolean | 'true' | 'false',
```

Comment:
It seems like there would be more places where we would need to make this change?

Reply:
This updates our types to match what React has for these properties. I don't think there are other places where we have this?